ingest/ingest-datasets.py

```diff
@@ -82,7 +82,7 @@ def create_document( self, entity ):
         if abstract:
             doc.update({"abstract": abstract})
 
-        publication_year = list(ds.objects(DCO.yearOfPublication))
+        publication_year = list(ds.objects(DCT.issued))
```

Comment:
This will be a date and not just a year.  You will need to use subsetting to extract just the year portion of the date.